relay-general/src/protocol/transaction.rs

```diff
@@ -20,6 +23,8 @@ pub enum TransactionSource {
     View,
     /// Named after a software component, such as a function or class name.
     Component,
+    /// The transaction name was updated to remove high cardinality parts.
```

Comment:
Probably a nit -- a "sanitized" transaction name doesn't necessarily require getting high cardinality parts removed. Currently, we are only going to support the removal feature, but let's not limit ourselves to that.
```suggestion
    /// The transaction name was updated to reduce the name cardinality.
```